pants-plugins/BUILD

```diff
@@ -0,0 +1,12 @@
+__defaults__(
+    all=dict(
+        resolve="pants-plugins",
+        skip_pylint=True,
+    )
+)
+
+# this adds a dependency on the pants libs using the version specified in pants.toml
+pants_requirements(
+    name="pants",
+    testutil=False,
```

Comment:
I set `testutil=False` (defaults to `True`), because I haven't used it so far. We can add it once we introduce some test(s) that use it.

see: https://www.pantsbuild.org/docs/reference-pants_requirements#codetestutilcode